packages/astroport-governance/src/assembly.rs

```diff
@@ -19,7 +19,9 @@ const MAX_DESC_LENGTH: usize = 1024;
 const MIN_LINK_LENGTH: usize = 12;
 const MAX_LINK_LENGTH: usize = 128;
 
-const ALLOWED_SIGNS: RangeInclusive<char> = '!'..='/';
+const ALLOWED_SIGNS: &str = "!\"#&()*+'-./";
+const LINK_REGEX: &str =
+    r"^(?:http(s)?://)?[\w.-]+(?:\.[\w\.-]+)+[\w\-\._~:/?#\[\]@!\$&'\(\)\*\+,;=.]+$";
```

Comment:
We dont need to escape special characters within ```[]``` parentheses.

Reply:
removed, thanks